python-checks/src/test/resources/checks/djangoRenderContext.py

```diff
@@ -0,0 +1,78 @@
+from django.shortcuts import render
+
+def using_locals_from_var(request):
+    username = "alice"
+    password = "p@ssw0rd"
+    context = locals()
+    return render(request, "my_template.html", context) # Noncompliant {{Use an explicit context instead of passing "locals()" to this Django "render" call.}} [[secondary=-1]]
```

Comment:
I could not find a proper way to test for secondary messages. I would like to verify that it is displayed correctly with the correct variable name.

Reply:
From the top of my head, I believe this can be done using the syntax:
`    #      <^^^^ {{Secondary message here.}}` as done for a normal primary location.
With a `>` or `<` sign to point towards the corresponding primary location.

Reply:
Perfect thank you!